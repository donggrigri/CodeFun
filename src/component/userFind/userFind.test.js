const firstData = {
    "createDate": "2019-01-14T08:45:43.952Z",
    "stars": [],
    "_id": "5c3c4c38a15a21061c50d818",
    "content": {
        "Title": "No Title",
        "userId": "t99480",
        "Htmlvalue": "awdasdz",
        "Cssvalue": "",
        "Jsvalue": "",
        "Modal": {
            "isModalOpen": false,
            "childComponent": null
        },
        "LibraryList": [],
        "UpdateNumber": 0,
        "AutoRunCheck": true,
        "LibIdNum": 0,
        "pending": false,
        "error": false,
        "ProjectId": null
    },
    "creator": "5c3c4c31a15a21061c50d817",
    "__v": 2
}
let secondData = {
    "createDate": "2019-01-14T09:00:21.256Z",
    "stars": [],
    "_id": "5c3c4fa87805780017566687",
    "content": {
        "Title": "새로만듬",
        "userId": "t99480",
        "Htmlvalue": "새로만들엇습니다",
        "Cssvalue": "",
        "Jsvalue": "",
        "Modal": {
            "isModalOpen": false,
            "childComponent": null
        },
        "LibraryList": [],
        "UpdateNumber": 0,
        "AutoRunCheck": true,
        "LibIdNum": 0,
        "pending": false,
        "error": false,
        "ProjectId": null
    },
    "creator": "5c3c4c31a15a21061c50d817",
    "__v": 0
};