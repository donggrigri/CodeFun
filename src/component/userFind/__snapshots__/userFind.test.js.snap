// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`userFind Component renders without crashing 1`] = `
<Fragment>
  <div
    className="userFindBox"
  >
    <div
      className="userFind__container"
    >
      <div
        className="userFind__Header"
      >
        <span
          className="backBtn"
          onClick={[Function]}
        >
          <FontAwesomeIcon
            border={false}
            className=""
            fixedWidth={false}
            flip={null}
            icon={
              Object {
                "icon": Array [
                  448,
                  512,
                  Array [],
                  "f060",
                  "M257.5 445.1l-22.2 22.2c-9.4 9.4-24.6 9.4-33.9 0L7 273c-9.4-9.4-9.4-24.6 0-33.9L201.4 44.7c9.4-9.4 24.6-9.4 33.9 0l22.2 22.2c9.5 9.5 9.3 25-.4 34.3L136.6 216H424c13.3 0 24 10.7 24 24v32c0 13.3-10.7 24-24 24H136.6l120.5 114.8c9.8 9.3 10 24.8.4 34.3z",
                ],
                "iconName": "arrow-left",
                "prefix": "fas",
              }
            }
            inverse={false}
            listItem={false}
            mask={null}
            pull={null}
            pulse={false}
            rotation={null}
            size="2x"
            spin={false}
            symbol={false}
            title=""
            transform={null}
          />
        </span>
        <Link
          replace={false}
          to="/"
        >
          CODE FUN
        </Link>
      </div>
      <div
        className="userFind__Body"
      >
        <div
          className="userFind__container__userName"
        >
          <div>
            Test
             ProjectList
          </div>
        </div>
        <div
          className="userFind__container__project"
        />
      </div>
    </div>
  </div>
</Fragment>
`;
